README.md

```diff
@@ -134,7 +137,18 @@ This repository is started to celebrate [Hacktoberfest 2020](https://hacktoberfe
 
   [![Star this repo](https://img.shields.io/github/stars/saileshbro/double-pendulum-simulation)](https://github.com/saileshbro/double-pendulum-simulation)
 
+## Open Source Projects
+
+### VS Code Extensions
```

Comment:
You can replace this with your project name

Reply:
Sure

Reply:
Thanks. and make sure you update the Table of contents too.